src/components/shareItemModal.js

```diff
@@ -48,9 +52,32 @@ class ShareItemModal extends Component {
           return user.id
         })
         const init_options = shared_users.map(user => {
+          const isStudent = user.roles ? user.roles.some(role => role.role === 'Student') : false
+          const isFaculty = user.roles ? user.roles.some(
+            role => role.role === 'FacultyMember'
+          ) : false
+          const extra_text = isStudent
+            ? `${
+                user.roles.find(role => role.role === 'Student').data
+                  .enrolmentNumber
+              } - ${
+                user.roles.find(role => role.role === 'Student').data.branch
+                  .name
+              }`
+            : isFaculty
+            ? `${
```

Comment:
this code was already written then why it is showing newly added?

Reply:
This was for rendering init_options, but i have removed it because its better to skip extra data for already selected users.